lightning/src/util/config.rs

```diff
@@ -397,6 +397,21 @@ pub struct ChannelConfig {
 	/// [`Normal`]: crate::chain::chaininterface::ConfirmationTarget::Normal
 	/// [`Background`]: crate::chain::chaininterface::ConfirmationTarget::Background
 	pub force_close_avoidance_max_fee_satoshis: u64,
+	/// Similar to [`Self::max_dust_htlc_exposure_msat`], but instead of setting a fixed maximum,
+	/// this sets a multiplier on the estimated high priority feerate (sats/KW, as obtained from
+	/// [`FeeEstimator`]) to determine the maximum allowed dust exposure. If this field is set to
+	/// `Some(value)`, then the maximum dust exposure in sats is calculated as:
+	/// `high_priority_feerate_per_kw * value / 1000`.
+	///
+	/// This allows the maximum dust exposure to automatically scale with fee rate changes. With
+	/// a fixed maximum, if the feerate increases significantly, then without a manually increase
+	/// to this maximum, the channel may be unable to send/receive HTLCs between the maximum dust
+	/// exposure and the new minimum value for HTLCs to be economically viable to claim.
+	///
+	/// Default value: `Some(5000)`.
+	///
+	/// [`FeeEstimator`]: crate::chain::chaininterface::FeeEstimator
+	pub max_dust_htlc_exposure_multiplier_thousandths: Option<u64>,
```

Comment:
Since this field is gone now that I switched to the enum and there's already a note in the main field's docs pointing out that with anchors the HTLC fees are not considered, I think this should be fine, but let me know if you think I should add to it